src/nls/root/strings.js

```diff
@@ -582,6 +583,14 @@ define({
     "CMD_RESTART_NODE"                          : "Restart Node",
     "CMD_SHOW_ERRORS_IN_STATUS_BAR"             : "Show Errors in Status Bar",
     "CMD_OPEN_BRACKETS_SOURCE"                  : "Open {APP_NAME} Source",
+    "ERROR_CREATING_LAUNCH_SCRIPT"              : "An erorr occured while creating {APP_NAME} launcher script at /usr/local/bin. Please refer to https://github.com/adobe/brackets/wiki/Command-Line-Arguments for troubleshooting. Reason: ",
+    "ERROR_CLTOOLS_RMFAILED"                    : "Failed to remove existing {APP_NAME} symlink at /usr/local/bin.",
```

Comment:
This is a MAC only change. We would not need strings for Win as the command line stuff is taken care of by our installer itself. And I have made change to the new shell api which would return an error if that API is called from platforms other than MAC